src/mumble/MainWindow.cpp

```diff
@@ -197,6 +195,7 @@ MainWindow::MainWindow(QWidget *p)
 
 	QObject::connect(this, &MainWindow::serverSynchronized, Global::get().pluginManager,
 					 &PluginManager::on_serverSynchronized);
+	QObject::connect(this, &MainWindow::serverSynchronized, this, &MainWindow::userStateChanged);
```

Comment:
Since it is (to my experience) a bit uncommon to connect a signal of a class to a slot of the same class, a comment explaining what this is for would be nice.